IDEAS/Buildings/Components/Shading/Interfaces/ShadingProperties.mo

```diff
@@ -70,6 +79,40 @@ record ShadingProperties
                       enable= (shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.Screen or
                                shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.OverhangAndScreen)));
 
+  parameter Modelica.SIunits.Length s(min=0)=0.01
+    "Vertical spacing between fins"
+    annotation(Dialog(group="Horizontal fins",
+                      enable= (shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.HorizontalFins or
+                               shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.OverhangAndHorizontalFins)));
+  parameter Modelica.SIunits.Length w(min=0)=0.01
+    "Fin width"
+        annotation(Dialog(group="Horizontal fins",
+                      enable= (shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.HorizontalFins or
+                               shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.OverhangAndHorizontalFins)));
+  parameter Modelica.SIunits.Length t(min=0)=0.01
+    "Fin thickness"
+        annotation(Dialog(group="Horizontal fins",
+                      enable= (shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.HorizontalFins or
+                               shaType==IDEAS.Buildings.Components.Shading.Interfaces.ShadingType.OverhangAndHorizontalFins)));
+  parameter Boolean use_betaInput = controlled
+    "=true, to use input for fin inclination angle"
+        annotation(Dialog(group="Horizontal fins",
+                      Evaluate = true,
+                      enable= false));
```

Comment:
@icupeiro I think it's better to set `final parameter Boolean use_betaInput=controlled` instead of `enable=false`?

Reply:
or is there a specific reason for doing it like this?